Katello/katello
# Fixes #35313 - CV Version Compare - add other content types

#### What are the changes introduced in this pull request?
Extended CV Version comparison feature to other content types, like Ansible collections and Python packages

#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?
1. Publish two versions so that either of them has all the 9 content types (RPM Packages, RPM Package groups, Files, Docker tags, Debs, Errata, Module streams, Python packages and Ansible collections). 
2. Select those two versions in the list of all versions 
3. Hit "Compare" button
4. Ensure that you are directed to the table with all 9 tabs for all the content types
5. Try switching the tabs and ensure that all of the tabs render required information
